### PR TITLE
Update hero hemkontakt

### PR DESCRIPTION
**Pull request by CyrilleKolle**
Link to ticket: https://fixa.itiden.se/project/1362/tickets/78

### CODE REVIEW

**Review by ricllrstn — CHANGES_REQUESTED** [review]
Looks good but change paddin-top from class 48 to 32. (Discussion for later: Do we need a min-height breakpoint that will release from 100vh.)

**Review by ricllrstn — COMMENTED** [review]
Looking good!

File: components/Animation/Background.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+export const SetBackgroundAnimation = styled.span<{
+  backgroundPosition: string;
+  // background: string;
+}>`
+  ${tw`p-2 -mt-2`}
+  display:inline;
```
Comment (reekris) [inline components/Animation/Background.tsx:10]: Remove this file from this PR. It is part of another PR

File: components/Layout/HeaderWide.tsx [hunk 2]
```diff
@@ -7,19 +7,19 @@ export const Wrapper = styled.div<{ viewportHeight: string }>`
   height: ${p => p.viewportHeight};
 
   @media screen and (min-width: 640px) {
-    ${tw`pt-48`}
-    min-height: 560px;
+    ${tw`pt-32`}
+    min-height: 560 - 40px;
```
Comment (reekris) [inline components/Layout/HeaderWide.tsx:11]: Instead of writing `560 - 40px` We should write `520px`

File: components/Layout/HeaderWide.tsx [hunk 3]
```diff
@@ -7,19 +7,19 @@ export const Wrapper = styled.div<{ viewportHeight: string }>`
...
     margin-top: -144px;
   }
 
   @media screen and (max-width: 639px) {
-    min-height: 480px;
+    min-height: 480 - 40px;
```
Comment (reekris) [inline components/Layout/HeaderWide.tsx:16]: Same as above

File: components/Layout/HeaderWide.tsx [hunk 4]
```diff
@@ -7,19 +7,19 @@ export const Wrapper = styled.div<{ viewportHeight: string }>`
...
     margin-top: -144px;
   }
 
   @media screen and (min-width: 768px) {
     ${tw`-mt-10`}
-    min-height: 820px; 
+    min-height: 820 - 40px;
```
Comment (reekris) [inline components/Layout/HeaderWide.tsx:22]: Same as above

**Review by reekris — CHANGES_REQUESTED** [review]
Some changes needed

**Review by reekris — APPROVED** [review]
Nice!

**Review by ricllrstn — APPROVED** [review]

**Pull request merged**